src/app/components/arlas-wui-root/geocoding/geocoding.component.ts

```diff
@@ -77,13 +78,15 @@ export class GeocodingComponent implements OnInit, AfterViewInit {
     }
 
     this.displayTable = true;
+    this.loading = true;
     this.previousSearch = this.searchFormControl.value;
     const geocodingSearch: GeocodingQueryParams = {
       q: this.searchFormControl.value,
       'accept-language': this.translateService.currentLang
     };
     this.geocodingService.findLocations(geocodingSearch).subscribe(r => {
       this.hasSearched = true;
+      this.loading= false;
```

Comment:
I suggest you also set the this.loading to false in case of an error

Reply:
Fixed